packages/zmq-eio/zmq-eio.5.3.0/opam

```diff
@@ -0,0 +1,41 @@
+opam-version: "2.0"
+synopsis: "Eio aware bindings to ZMQ"
+maintainer: ["Anders Fugmann <anders@fugmann.net>"]
+authors: ["Anders Fugmann"]
+license: "MIT"
+homepage: "https://github.com/issuu/ocaml-zmq"
+bug-reports: "https://github.com/issuu/ocaml-zmq/issues"
+depends: [
+  "dune" {>= "2.7"}
+  "ocaml" {>= "4.04.1"}
+  "zmq" {= version}
+  "eio" {>= "0.10"}
```

Comment:
Should this be `>= 0.10 < 1.0`? In case of breaking changes.

Reply:
We try to avoid preventive upper bounds: if we see a breakage when eio 1.0 is released, we will add the upper bound manually.

In fact, this has already been built and tested by the CI with eio 1.0